dev-config.edn

```diff
@@ -92,17 +92,7 @@
  :enable-catalogue-tree true
  :enable-save-compaction true
  :enable-autosave true
- :extended-logging-routes ["/api/catalogue-items/create"
-                           "/api/catalogue-items/edit"
-                           "/api/categories/create"
-                           "/api/categories/edit"
-                           "/api/forms/create"
-                           "/api/forms/edit"
-                           "/api/users/create"
-                           "/api/users/edit"
-                           "/api/user-settings/edit"
-                           "/api/workflows/create"
-                           "/api/workflows/edit"]
+ :extended-logging true
```

Comment:
For consistency we could name this `:enable-extended-logging` or consider alternatives like `:audit-log-request-and-response` or `:enable-audit-logging-of-request-and-response`.

Reply:
Definitely should be prefixed with `enable`.

But I think the other two alternatives are not descriptive of what it is meant to do. It is not necessary to log at this level for all requests, as most are persisted in other ways that result in immutable entities, as such, there is always an audit trail that points to a specific user who committed a specific change.

The entities can be divided into:
1. Immutable entities that cannot be changed, but support copy and modify.
2. Mutable entities that can be changed.

Extended logging is only relevant for the second group.

Reply:
naming addressed in [33a93e8](https://github.com/CSCfi/rems/pull/3187/commits/33a93e8622f6b11d1c6556a90d6eb5ecec357e82)